src/tox_gh_actions/plugin.py

```diff
@@ -35,6 +35,20 @@ def tox_configure(config):
     config.envlist_default = config.envlist = envlist
 
 
+@hookimpl
+def tox_runtest_pre(venv):
```

Comment:
I think I also want the env create/installdeps/inst/installed grouped

see: https://github.com/twisted/ldaptor/pull/206/checks?check_run_id=1205103588#step:8:1698